js/gated-content/src/components/video/VideoTeaser.vue

```diff
@@ -68,7 +66,8 @@ export default {
       return this.video.attributes['field_gc_video_media.thumbnail'].image_style_uri[0].gated_content_teaser;
     },
     duration() {
-      return moment.duration(this.video.attributes.field_gc_video_duration, 'seconds').format('m [minute]');
+      return `${Math.floor(dayjs.duration(this.video.attributes.field_gc_video_duration, 'seconds')
+        .asMinutes())} minutes`;
```

Comment:
Can this correctly display `1 minute`?

Reply:
@retif we need to think about plural response.
Maybe dayjs has it on board?

Reply:
https://day.js.org/docs/en/plugin/plural-get-set